<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="A_lbl">
          <property name="text">
           <string>A: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="x1_lbl">
          <property name="text">
           <string>x1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x1_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="y1_lbl">
          <property name="text">
           <string>y1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y1_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="B_lbl">
          <property name="text">
           <string>B: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="x2_lbl">
          <property name="text">
           <string>x2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x2_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="y2_lbl">
          <property name="text">
           <string>y2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y2_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="C_lbl">
          <property name="text">
           <string>C: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="x3_lbl">
          <property name="text">
           <string>x3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x3_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="y3_lbl">
          <property name="text">
           <string>y3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y3_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="Point_lbl">
          <property name="text">
           <string>Точка:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="x0_lbl">
          <property name="text">
           <string>x0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="x0_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="y0_lbl">
          <property name="text">
           <string>y0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="y0_sb">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="angle_lbl">
          <property name="text">
           <string>Угол:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="angle_sb">
          <property name="minimum">
           <number>-360</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="draw_button">
        <property name="text">
         <string>Нарисовать фигуру</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="turn_button">
        <property name="text">
         <string>Поворот</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="plot_widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
